S11-modules/perl6lib.t

```diff
@@ -5,11 +5,13 @@ use Test::Util;
 
 plan 1;
 
-# TODO Replace PERL6LIB with RAKULIB when supported
-is_run 'BEGIN { BEGIN { q{S11modulesRakuLibTest.pm6}.IO.spurt(q{package { say q{all your base} }}); %*ENV<PERL6LIB>=qq{}; }; use S11modulesRakuLibTest }',
+# Test deprecated environment variable PERL6LIB
+# TODO remove this file when PERL6LIB is removed
+is_run 'BEGIN { BEGIN { q{S11modules-PERL6LIB-Test.rakumod}.IO.spurt(q{package { say q{all your base} }}); %*ENV<PERL6LIB>=qq{}; };\
+        use S11modules-PERL6LIB-Test }',
```

Comment:
Why is this broken onto a new line?

Reply:
So one can see all the text in a terminal with non-folding lines.